emukit/core/event_handler.py

```diff
@@ -0,0 +1,11 @@
+class EventHandler(list):
+    """
+    A list of callable objects. Calling an instance of this will cause a call to each item in the list in ascending
```

Comment:
Let's link the SO answer here. I know SO does not demand it, but attribution does not cost anything and is a nice thing to do.

Reply:
Done